src/main/java/uk/co/ramp/event/VirusEventProcessor.java

```diff
@@ -50,8 +61,14 @@ public ProcessedEventResult processEvent(VirusEvent event) {
               .time(event.time() + deltaTime)
               .build();
 
-      List<AlertEvent> alertEvents = checkForAlert(event.id(), event.time());
-
+      List<AlertEvent> alertEvents;
+      if (event.nextStatus() == VirusStatus.SYMPTOMATIC
+          && thisCase.reportingCompliance() > distributionSampler.uniformBetweenZeroAndOne()) {
```

Comment:
Hi @sam-brett, there is already a check for if the person is symptomatic within the `checkForAlert` call (in AlertChecker.java). So I think that part of the conditional statement can be removed, so that we ensure code isn't duplicated

```suggestion
      if (thisCase.reportingCompliance() > distributionSampler.uniformBetweenZeroAndOne()) {
```